### PR TITLE
Adding cert verification parameter

### LINKED ISSUE #134: [RFE] for ansible-ipi-install: Ability to use redfish instead of ipmi
Currently the install-config.yaml only uses ipmi. We want the ability to use redfish or ipmi.

### PR DESCRIPTION
**Pull request by DirectedSoul1**
# Description

Adding a Refish cert verification parameter to the documentation section. 

Fixes #134 

Preview at: https://deploy-preview-479--baremetal-deploy.netlify.app/4.4/deployment#additional-install-config-parameters_ipi-install-prerequisites

## Type of change

Please select the appropriate options:

- [x] This change requires a documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [x ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc [hunk 1]
```diff
@@ -16,6 +16,7 @@ and the `bmc` `address` parameter for the `install-config.yaml` file.
 provisioning network. ({product-title} 4.4 only)
 | `hosts` |  | Details about bare metal hosts to use to build the cluster.
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
+| `disableCertificateVerification` | | This parameter is needed when we use `redfish` instead of `ipmi` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses or `False` if you have a Certificate Authority.
```
Comment (rlopez133) [inline documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc:19]: This parameter is needed when using `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value must be set to `True` when using self-signed certificate for BMC addresses.

Comment (DirectedSoul1) [reply documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc]: Added the same as suggested.

File: documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc [hunk 2]
```diff
@@ -16,6 +16,7 @@ and the `bmc` `address` parameter for the `install-config.yaml` file.
 provisioning network. ({product-title} 4.4 only)
 | `hosts` |  | Details about bare metal hosts to use to build the cluster.
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
+| `disableCertificateVerification` | | This parameter is needed when we use `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses or `False` if you have a Certificate Authority.
```
Comment (rlopez133) [inline documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc:19]: Lets leave the or False part out. Lets put False in the default column so it should look like

| `disableCertificateVerification` | False | This parameter is needed when we use `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses.

Comment (DirectedSoul1) [reply documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc:19]: Yeah, sounds good.

Comment (rlopez133) [reply documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc]: Looks good, squash down to 1 commit?

Comment (DirectedSoul1) [reply documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc]: yeah, sure squashing it now.

**Pull request merged**